src/home_robot/home_robot/agent/objectnav_agent/objectnav_agent.py

```diff
@@ -251,7 +253,8 @@ def _preprocess_obs(self, obs: Observations):
         obs_preprocessed = torch.cat([rgb, depth, semantic], dim=-1).unsqueeze(0)
         obs_preprocessed = obs_preprocessed.permute(0, 3, 1, 2)
 
-        curr_pose = np.array([obs.gps[0], -obs.gps[1], obs.compass[0]])
+        # curr_pose = np.array([obs.gps[0], -obs.gps[1], obs.compass[0]])
+        curr_pose = np.array([obs.gps[0], obs.gps[1], obs.compass[0]])
         """
         curr_pose = obs2xyt(obs.base_pose)
         curr_pose[1] = -curr_pose[1]  # agent y axis is flipped, not sure why
```

Comment:
Remove comments and isolate into a conversion function.

Reply:
this shouldnt have a conversion function